Calculator/src/main/java/com/microsoft/device/display/samples/calculator/ui/pages/components/Buttons.kt

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.device.display.samples.calculator.ui.pages.components
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.microsoft.device.display.samples.calculator.R
+import com.microsoft.device.display.samples.calculator.models.Action
+import com.microsoft.device.display.samples.calculator.models.Equation
+import com.microsoft.device.display.samples.calculator.models.actionStringEquivalent
+import com.microsoft.device.display.samples.calculator.models.equationStringEquivalent
+import com.microsoft.device.display.samples.calculator.ui.pages.calculatorModel
+
+@Composable
+fun EquationButton(eq: Equation, color: Color = MaterialTheme.colors.primary) {
+    ButtonLayout(
+        color = color,
+        content = equationStringEquivalent(eq)
+    ) { calculatorModel.updateEquation(eq) }
+}
```

Comment:
optional style suggestion: since this lambda is for an onClick method and not composable content, it may be more clear to explicitly name the parameter
```suggestion
@Composable
fun EquationButton(eq: Equation, color: Color = MaterialTheme.colors.primary) {
    ButtonLayout(
        color = color,
        content = equationStringEquivalent(eq)
        onClick = { calculatorModel.updateEquation(eq) }
    ) 
}
```